test/app/languageforge/lexicon/newLexProject/e2e/lexicon-new-project.spec.js

```diff
@@ -363,7 +363,6 @@ describe('E2E testing: New Lex Project wizard app', function () {
       expect(page.backButton.isDisplayed()).toBe(false);
       expect(page.initialDataPage.browseButton.isDisplayed()).toBe(true);
       expect(page.progressIndicatorStep3Label.getText()).toEqual('Verify');
-      page.expectFormIsNotValid();
```

Comment:
I looked into this pretty deeply for a good while before deciding removing the assertion was the best course of action. It appears the form validation logic considers there to be three states: valid, invalid, and somewhere in between (for the condition of no input perhaps?). When one progresses quickly past the "project name" state to the "import project" state the form is marked valid without ever uploading anything (it's possible to skip the step though, so it makes sense to consider it valid).

From the code it appears this was the intended behavior, but it's not what the tests look for, and because validation isn't run until there is user input, it's not usually the behavior. I'm not sure why this wasn't happening before (on my machine it even happens on master, whereas it wasn't on my previous dev machine). It's also inconsistent (it's a race condition).

There's basically the same issue below, where it again asserts that the form is not valid but that it's valid enough to move on.